BAZProjectUnitTests/TrendingMoviesViewControllerTests.swift

```diff
@@ -40,10 +27,11 @@ final class TrendingMoviesViewControllerTests: XCTestCase {
     func test_NumberOfRowsInSection_ShouldBeZero() {
         //Given
         let movies = [Movie]()
+        let viewModel = TrendingMoviesViewModelMock(movies: movies, remoteData: TrendingRemoteDataMock())
+        let sut = TrendingMoviesViewController(viewModel: viewModel)
```

Comment:
Tomando el mismo ejemplo del comentario anterior
```suggestion
        let sut = makeSUT()
```